posthog/hogql_queries/insights/trends/trends_query_runner.py

```diff
@@ -812,3 +812,10 @@ def _trends_display(self) -> TrendsDisplay:
             display = self.query.trendsFilter.display
 
         return TrendsDisplay(display)
+
+    def apply_dashboard_filters(self, *args, **kwargs) -> RunnableQueryNode:
+        # Remove any set breakdown limit for display on the dashboard
```

Comment:
After https://github.com/PostHog/posthog/pull/21720, this will 100% need a `super().apply_dashboard_filters()`, but I'll push a change to add this here when the other PR's in